src/main/java/org/synyx/urlaubsverwaltung/settings/TimeSettings.java

```diff
@@ -0,0 +1,44 @@
+package org.synyx.urlaubsverwaltung.settings;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+
+import static java.util.Optional.ofNullable;
+
+@Embeddable
+public class TimeSettings {
+
+    @Column(name = "timezoneid")
+    private String timeZoneId;
+
+    @Column(name = "workDayBeginHour")
+    private Integer workDayBeginHour = 8; // NOSONAR
+
+    @Column(name = "workDayEndHour")
+    private Integer workDayEndHour = 16; // NOSONAR
+
+    public String getTimeZoneId() {
+
+        return ofNullable(timeZoneId).orElse("Europe/Berlin");
```

Comment:
hab nicht geschaut. Würde ich direkt in die db schreiben per script. Sonst wäre erstmal nichts in der db gesetzt obwohl diesen den Status beschreibt.